src/main/kotlin/no/skatteetaten/aurora/gorg/extensions/HasMetadata.kt

```diff
@@ -3,12 +3,20 @@ package no.skatteetaten.aurora.gorg.extensions
 import io.fabric8.kubernetes.api.model.HasMetadata
 import java.lang.IllegalStateException
 import java.time.Instant
+import mu.KotlinLogging
 
 const val REMOVE_AFTER_LABEL = "removeAfter"
 const val TERMINATING_PHASE = "Terminating"
 
-fun HasMetadata.removalTime(): Instant {
+private val logger = KotlinLogging.logger {}
+
+fun HasMetadata.removalTime(): Instant? {
     return this.metadata.labels[REMOVE_AFTER_LABEL]?.let {
+        if (it == "null") {
+            logger.info { "Not a valid timestamp for removeAfter kind=${this.kind} namespace=${this.metadata.namespace} name=${this.metadata.name}" }
```

Comment:
null som returneres her vil vel trigge throw IllegalStateException? trenger vi å info-logge dette?

Reply:
nei det vil den vel ikke? Jeg returnerer utifra removalTime. Jeg ønsker ikke at dette skal kaste en IllegalStateException. Men vi kan diskutere om det er hensiktsmessig at IllegalStateException i det hele tatt blir kastet. Jeg synes kanskje det hadde vært bedre å logge en warning også ignorert hele objektet. Hvis det skjer en feil så "stopper" hele gorg basically

Reply:
Jeg fikk for meg at det var kjedet `let == null --> throw` og `let != null --> let-result == null --> throw`.
Kan du legge til en testcase hvor removeAfter har ugyldig verdi?

Jeg synes også det er mer fornuftig å logge warning,  ignorere objektet og håndtere resterende objekter med gyldig removeAfter verdi.

Reply:
Jeg kan legge til et testCase :) 

Jeg kan forandre til slik vi er enige om. Da vil det bli mye klarere hva som skjer her i tillegg tenker jeg